doc/users/next_whats_new/diverging_colormaps.rst

```diff
@@ -0,0 +1,24 @@
+Dark-mode diverging colormaps
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Three diverging colormaps have been added: "berlin", "managua", and "vanimo".
+They are dark-mode diverging colormaps, with minimum lightness at the center,
+and maximum at the extremes. These are taken from F. Crameri's Scientific
+colour maps version 8.0.1 (DOI: https://doi.org/10.5281/zenodo.1243862).
+
+
+.. plot::
+    :include-source: true
+    :alt: Example figures using "imshow" with dark-mode diverging colormaps on positive and negative data. First panel: "berlin" (blue to red with a black center); second panel: "managua" (orange to cyan with a dark purple center); third panel: "vanimo" (pink to green with a black center).
```

Comment:
We don't lint rst files, but pretty sure this should be wrapped.  I think as long as you indent the plot directive should still be OK.

Reply:
I wasn't sure whether breaking the line would send me out of the `alt` variable. I checked the previous what's new files and they seem to go on without breaking the line: https://raw.githubusercontent.com/matplotlib/matplotlib/ed56f95cb6f5b54525687dabbb0e660bafec4d94/doc/users/prev_whats_new/whats_new_3.9.0.rst

Reply:
I'm not sure either, and maybe it was done that way on purpose.  But if not, then it would be nice to have wrapping

Reply:
Unfortunately I was unable to compile the docs locally due to some dependency issues, so I cannot check. Sorry.

Reply:
Thats why we have CI

Reply:
That didn't work, I reverted.